wasp/apps/settings.py

```diff
@@ -37,7 +37,9 @@ def __init__(self):
         self._dd = wasp.widgets.Spinner(20, 60, 1, 31, 1)
         self._mm = wasp.widgets.Spinner(90, 60, 1, 12, 1)
         self._yy = wasp.widgets.Spinner(160, 60, 20, 60, 2)
-        self._settings = ['Brightness', 'Notification Level', 'Time', 'Date']
+        self._von = wasp.widgets.Checkbox(3, 40, 'Vibrate')
```

Comment:
I'm not sure that this checkbox is needed? If you want to disable Notifications then set them to Silent with the slider.